configuration/drugs/drug2.csv

```diff
@@ -0,0 +1,197 @@
+Uuid,Void/Retire,Name,Concept Dosage Form,Concept Drug,Strength,_version:1

```

Comment:
Missing column for the order sequence, This sheet should be processed after the drugs concept sheet has been processed. So the Order sequence should be higher. I will proceed and fix this @kraigochieng